config.toml

```diff
@@ -89,3 +89,13 @@ params.repositories = [
     "taskcluster/slugid",
 ]
 tags = ["taskcluster"]
+
+
+[checks.secrets.writeable]
+description = "Secret Service writable"
+module = "checks.taskcluster.write_secrets"
+params.root_url = "${TASKCLUSTER_ROOT_URL}"
+params.client_id = "${TASKCLUSTER_CLIENT_ID}"
+params.access_token = "${TASKCLUSTER_ACCESS_TOKEN}"
+params.certification = "${TASKCLUSTER_CERTIFICATE}"
```

Comment:
The parameter is called `certificate` but using `access_token` is enough so you can get rid of it